packages/appium/scripts/autoinstall-extensions.js

```diff
@@ -227,10 +181,7 @@ async function checkAndInstallExtension({
 if (require.main === module) {
   main().catch((e) => {
     log(e);
-    if (process.env.CI) {
-      log('Detected CI environment, exiting with code 1');
-      process.exitCode = 1;
-    }
+    process.exitCode = 1;
```

Comment:
tbh I'm happy about exiting nonzero on a failure, but it's the only way an end-user is going to know, because otherwise the output from this script is squelched by default.